.github/CODEOWNERS

```diff
@@ -1,2 +1,4 @@
-seed/seed.json   @brave/uplift-approvers
+# The order is important. The last matching pattern takes the most precedence.
+
 *                @brave/griffin-maintainers
+/seed/seed.json   @brave/uplift-approvers
```

Comment:
Shouldn't the @ come left by 1 space?

Reply:
yes, it looked wrong. Instead I've added comments and separated entries, so we don't need to sync offsets on all lines at all.